[Translation] [Loco] Ability to configure value of `status` query-variable (#58072)
symfony/symfony: src/Symfony/Component/Translation/Bridge/Loco/CHANGELOG.md

CHANGELOG
=========

7.2
---

> * Added ability to configure value of `status` query-variable when fetching translations from loco API.

Reviewer: ```suggestion
 * Add support for the `status` query parameter of Loco translation API
```
